### PR TITLE
fix: DiscTrapezoidBounds vertices

### PR DESCRIPTION
**Pull request by XiaocongAi**
This PR fixes the vertices calculation of a trapezoid in the local  frame (before the transformation).

### CODE REVIEW

**Comment by XiaocongAi** [comment]
Hi @asalzburger , could you help have a look?

File: Core/src/Surfaces/DiscTrapezoidBounds.cpp
Comment (paulgessinger) [inline Core/src/Surfaces/DiscTrapezoidBounds.cpp:1]: I don't fully follow the geometrical relationships here, unfortunately. 
Which geometric point does `ymin` reference? Is the problem that before the vertices were not centered around the center of the trapezoid area itself?

Comment (XiaocongAi) [reply Core/src/Surfaces/DiscTrapezoidBounds.cpp:1]: image

I made a simple sketch to illustrate the `ymin`. Yes, a trapezoid before transformation should be centered at (0,0), which is different from a tube. Obviously, the current vertices are those when the trapezoid is already translated as shown in the sketch.

Comment (paulgessinger) [reply Core/src/Surfaces/DiscTrapezoidBounds.cpp:1]: Ok, got it.

**Review by paulgessinger — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :white_check_mark: Athena integration test results
### :white_check_mark: All tests successful

|status|job|
|------|---|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143610) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsWorkflow.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143609) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateTracks.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143608) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsCoreSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143607) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateActsSpacePoints.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143606) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143605) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateOrthogonalSeeds.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143604) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsValidateClusters.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143603) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsPersistifyEDM.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143602) |
|:green_circle: | [run_ci_tests: ../athena/AtlasTest/CITest/test/ActsKfRefitting.sh](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143601) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsExtrapolationAlgTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143600) |
|:green_circle: | [run_ci_tests: python3 ../athena/Tracking/Acts/ActsGeometry/test/ActsITkTest.py](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143599) |
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143598) |
|:green_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143597) |
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143596) |
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143595) |
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/33143594) |

**Pull request merged**